Formula/s/service-weaver.rb

```diff
@@ -13,6 +13,11 @@ class ServiceWeaver < Formula
     end
   end
 
+  livecheck do
```

Comment:
```suggestion
  # Upstream does not create releases for all tags and it's currently unclear
  # whether we should use stable tags that don't have a release, so the
  # `GithubLatest` strategy is necessary.
  livecheck do
```

Since the `stable` URL isn't a release asset (it's a tag archive), we need to add a comment to explain why we're using the `GithubLatest` strategy.